config/dapp1@dapp.priv

```diff
@@ -0,0 +1 @@
+43eeb17f0bab10dd51ab70983c25200a1742d31b3b7b54c38c34d7b827b26eed
```

Comment:
Is it correct private key for public `0000000000000000000000000000000000000000000000000000000000000000`?

Reply:
I think yes, it's taken from IPJ default keypair. Anyways, it might be changed later.